### ISSUE TITLE
Component-style tooltip displaying incorrectly when nested in another component

### ISSUE THREAD

**Issue by digitalnomad624** [FlutterFlow/flutterflow-issues#4848]
### Can we access your project?

- [X] I give permission for members of the FlutterFlow team to access and test my project for the sole purpose of investigating this issue.

### Current Behavior

When a component-style tooltip is used within a parent component, and component parameters are passed from parent to child, the tooltip doesn't display correctly on the page containing the parent component. In my case only the first text field is shown, and it's formatted as a text-style tooltip.

### Expected Behavior

Nested tooltip components should be visible as designed when triggered, regardless of how many levels deep they're nested (or at least 2 levels). 

## Create the component to hold the tooltip text
- Create a new component containing text fields for title and description
- Set up component parameters to populate the text fields
- This is the 'tooltipText' component in the screenshots

## Create the parent component
- Create a new component (or use a template) 
- Add parameters for the title and description
- Add a tooltip to the component and configure the inputs to use the tooltipTitle and tooltipDescription parameters
- This is the 'dashboardKpiWithTrend' component in the screenshots

## Create a page
- Add a new page
- Add the parent component to the page
- Fill out the component parameters for title and description

## Test the app
- The tooltip will show only the title, formatted as a text tooltip, not as the component

### Reproducible from Blank

- [X] The steps to reproduce above start from a blank project.

### Bug Report Code (Required)

IT5XzMmBw5RgocNG0KqNKe9aniksJkQ9U400ltZxYw81GJTnPLAcPPfET3NaTbykTHNcOWCI8TkKw97LvfzhDsctPiitbqJNzLhucBXJez6QRIyvCbiOe3xtHphTN2WHzbbRkQpTNbFociU+2U6IIeeXUXLYNJO3OTERM/ykOpLM+HSRFTLOXn0Jg1ZSfCvr

## Dashboard KPI component in test mode with incorrect tooltip
![2024-11-07_20-30-02](https://github.com/user-attachments/assets/65da0249-a7d8-4d7b-9daa-820c49d83d29)

## tooltipText component
![2024-11-07_20-32-57](https://github.com/user-attachments/assets/15119e12-7800-4316-8e4e-5816d1b2822d)

## dashboardKpiWithTrend (parent) component
![2024-11-07_20-32-44](https://github.com/user-attachments/assets/645460ba-8935-42ff-bcd8-1f9241fde5f4)

## Dashboard page passing the title and text to the parent component
![2024-11-07_21-09-30](https://github.com/user-attachments/assets/7ea64ba6-46bf-4b31-beaa-4d22bd96d711)

## tooltipText component used in a tooltip directly on the page
To make sure the tooltipText component works properly, I added it to a tooltip directly on the page. It works as expected.
![2024-11-07_20-30-22](https://github.com/user-attachments/assets/148ecbc4-ddcb-4a0f-a249-e6fd5446a2b6)

### Environment

```markdown
- FlutterFlow v5.0.9 released October 25, 2024 
- Flutter version is 3.24.2
- Platform: Mac app
- Browser name and version: Chrome Version 130.0.6723.117 (Official Build) (x86_64)
- Operating system and version affected: macOS 14.6.1
```

### Additional Information

Not being able to use the new component-style tooltip feature inside another component isn't a showstopper - we can still use parameters to fill in the text-style tooltips - but the new component-style tooltips look great and the additional formatting options will help my users better, and more quickly, understand the additional information we want to get across with a tooltip.

**Comment by Alezanello** [comment]
Hello!

I tried to replicate the issue you described but was unable to do so. It’s possible I may have missed some steps. Here’s what I did:

1. Created Component A with two text fields, which are populated by two parameters (title & body).
2. Created Component B and added a Tooltip widget containing a component. I selected Component A as the component and passed the two parameters to fill in the title and body fields.
3. Created Page A, inserted Component B, and passed the parameters needed to fill Component A's title and body.

This setup results in the information for the title and body being sent from Page A to Component B, and then to Component A for display.<br><br>Public Project: [https://app.flutterflow.io/project/qa-2991-zspgcs](https://app.flutterflow.io/project/qa-2991-zspgcs)

<img src="https://uploads.linear.app/3241089d-736a-4422-900b-051e72accdbe/dcecdb64-00fa-404e-8c0f-d0b8b69bed35/c0a7bb83-de18-411d-91c3-5591ae5d7d51?signature=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJwYXRoIjoiLzMyNDEwODlkLTczNmEtNDQyMi05MDBiLTA1MWU3MmFjY2RiZS9kY2VjZGI2NC0wMGZhLTQwNGUtOGMwZi1kMGI4YjY5YmVkMzUvYzBhN2JiODMtZGUxOC00MTFkLTkxYzMtNTU5MWFlNWQ3ZDUxIiwiaWF0IjoxNzMxMzU4OTkzLCJleHAiOjMzMzAxOTE4OTkzfQ.QIzsimw2R6kHwx6uc1AMic9zNQM--w5QD34o2gwHV8k " alt="Tooltip.gif" width="1421" height="890" />
